Hyper/Window.cs

```diff
@@ -225,5 +223,10 @@ public void RegisterCallbacks()
         _context.RegisterKeys(new List<Keys> { Keys.Escape });
 
         _context.RegisterKeyDownCallback(Keys.Escape, Close);
+        _context.RegisterKeyDownCallback(Keys.T, () =>
+        {
+            if (_console.IsAlive) StopDebugThread();
```

Comment:
what should be the visible effect of this action?

Reply:
it should free the thread that waits for console input